packages/vuetify/src/labs/VDataTable/composables/sort.ts

```diff
@@ -7,16 +7,20 @@ import { getObjectValueByPath, propsFactory } from '@/util'
 
 // Types
 import type { InjectionKey, PropType, Ref } from 'vue'
-import type { InternalItem } from '@/composables/items'
-import type { DataTableCompareFunction, DataTableItem, InternalDataTableHeader } from '../types'
+import type { DataTableCompareFunction, InternalDataTableHeader } from '../types'
 
 export const makeDataTableSortProps = propsFactory({
   sortBy: {
     type: Array as PropType<SortItem[]>,
     default: () => ([]),
   },
+  customKeySort: Object as PropType<Record<string, DataTableCompareFunction>>,
   multiSort: Boolean,
   mustSort: Boolean,
+  locale: {
```

Comment:
Should this take the locale default value into account?

Reply:
Oh, yes. We should probably remove that prop and use locale composable instead. I don't think it's a good idea to have 3 different ways of setting the locale. Global + v-locale-provider using composable should be enough.